aiida/cmdline/commands/cmd_calcjob.py

```diff
@@ -90,23 +90,31 @@ def calcjob_inputcat(calcjob, path):
     """
     from aiida.plugins.entry_point import get_entry_point_from_class
 
+    # Get path from the given CalcJobNode
     if path is None:
-
         path = calcjob.get_option('input_filename')
 
-        if path is None:
-            cls = calcjob.__class__
-            _, entry_point = get_entry_point_from_class(cls.__module__, cls.__name__)
-            echo.echo_critical('{} does not define a default input file. Please specify a path explicitly'.format(
-                entry_point.name))
+    # Try "default" input_filename value
+    warn = False
+    if path is None:
+        path = "aiida.in"
+        warn = True
 
     try:
         content = calcjob.get_object_content(path)
     except Exception as exception:  # pylint: disable=broad-except
-        echo.echo_critical('failed to get the content of file `{}`: {}'.format(path, exception))
+        # No path available or is incorrect
+        cls = calcjob.__class__
+        _, entry_point = get_entry_point_from_class(cls.__module__, cls.__name__)
+        echo.echo_critical('{} does not define a default input file. Please specify a path explicitly.\n'
+                           'Exception: {}'.format(entry_point.name, exception))
     else:
         echo.echo(content)
 
+        if warn:
+            echo.echo_warning('Attribute/Option "input_filename" not found. Used default "aiida.in".\n'
```

Comment:
Correct. The logic here being that if the 'input_filename' option is specified by the plug-in, it would never try to look for `aiida.in`, therefore there would be no need to warn.
But if a user thought he/she had already set this option, this may be helpful in finding out it wasn't set at all, and has instead defaulted to `aiida.in`.

I could see why the warning would be annoying with prolonged usage, and perhaps a note should be added to the critical message above saying something like "also tried the default 'aiida.in' without luck".